e2e/tests/ui/cohort-ui.spec.ts

```diff
@@ -87,4 +86,75 @@ describe('Cohort UI Test', () => {
     // Changes are discarded, back to the Data page.
     await dataPage.waitForLoad();
   });
+
+  test('Cohort search by code and description', async () => {
+    // Find and open one workspace.
+    const workspaceName = await openWorkspace(page);
+    if (!workspaceName) {
+      return;
+    }
+
+    const dataPage = new WorkspaceDataPage(page);
+    await dataPage.clickAddCohortsButton();
+
+    // Landing in Build Cohort Criteria page.
+    const cohortBuildPage = new CohortBuildPage(page);
+    await cohortBuildPage.waitForLoad();
+
+    const group1 = cohortBuildPage.findIncludeParticipantsGroup('Group 1');
+    await group1.addCriteria([MenuOption.Procedures]);
+    await waitWhileLoading(page);
+
+    const resultsTable = await group1.getSearchResultsTable();
+    const originalRowNum = (await resultsTable.getRows()).length;
+    console.log(`originalRowNum: ${originalRowNum}`);
+
+    // Search by description.
+    const description = 'Surgical pathology';
+    await group1.searchCriteria(description);
+    await searchAndVerifyResults(resultsTable, description);
+
+    // Search by code.
+    const code = '128927009';
+    await group1.searchCriteria(code);
+    await searchAndVerifyResults(resultsTable, code, 5);
+
+    // Search by Concept ID. DOESN'T WORK!
+    const conceptId = '2213280';
+    await group1.searchCriteria(conceptId);
+    const resultsRow = await resultsTable.getRowCount();
+    expect(resultsRow).toEqual(0);
+  });
+
+  async function openWorkspace(page: Page): Promise<string | null> {
+    // Find all workspaces that are older than 30 min.
+    const allWorkspaceCards = await findAllCards(page, 1000 * 60 * 30);
+    // Don't create new workspace if none found.
+    if (allWorkspaceCards.length === 0) {
+      logger.info('Cannot find a suitable existing workspace (created at least 30 min ago). Test end early.');
+      return null;
+    }
+
+    // Open any one workspace.
+    const aWorkspaceCard = fp.shuffle(allWorkspaceCards)[0];
+    const workspaceName = aWorkspaceCard.getWorkspaceName();
+    await aWorkspaceCard.clickWorkspaceName();
+    3;
+    return workspaceName;
+  }
+
+  // Verify search results contain search word.
+  async function searchAndVerifyResults(resultsTable: Table, keyword: string, columnIndx = 1): Promise<void> {
+    const searchResultsRowNum = (await resultsTable.getRows()).length;
+    console.log(`searchResultsRowNum: ${searchResultsRowNum}`);
+
+    expect(searchResultsRowNum).toBeGreaterThan(1);
```

Comment:
This is failing for me locally when searching by the code '128927009' since it only returns one result. Not sure why it would pass the CI run since they're both running against test api and should get the same search results.

Searching by the keywords 'Surgical pathology' passes for me locally, but the names of results are not guaranteed  to contain the search terms so this could easily fail if the data/order changes or for a different keyword search. For example, the 8th result of this search does not contain the words 'Surgical pathology':
<img width="1000" alt="Screen Shot 2021-09-07 at 10 55 06 PM" src="https://user-images.githubusercontent.com/40036095/132444061-19ec5afe-6d1f-4162-9ab8-33bcb3a62f32.png">

I think this type of validation of results is hard to do on the front end, but it also may be unnecessary since I believe we have coverage for this in the api tests (@freemabd correct me if that's not the case). It may be enough to just test that we're getting any results for valid searches and no results for invalid searches.